yaml/pois.yaml

```diff
@@ -233,7 +233,7 @@ global:
         - natural: [ cave_entrance, peak, volcano, geyser, hot_spring, rock, saddle,
             stone, spring, tree, waterfall ]
         - power: [ pole, tower, generator ]
-        - public_transport: [ platform, stop_area ]
+        - public_transport: [ platform, station, stop_area ]
```

Comment:
big stations with POIs visible at zoom 10 should have a name, please remove station here.

Reply:
Fixed in https://github.com/tilezen/vector-datasource/pull/1756/commits/628980cd96284e46f76594d0c132f746f34f3f0c.